apps/api/schedule.py

```diff
@@ -169,6 +170,69 @@ def post(self, proposal_type):
         return [t.id for t in res]
 
 
+class ProposalC3VOCPublishingWebhook(Resource):
+    method_decorators = {"post": [_require_video_api_key]}
+
+    def post(self):
+        if not request.is_json:
+            abort(415)
+
+        payload = request.get_json()
+
+        try:
+            conference = payload["fahrplan"]["conference"]
+            proposal_id = payload["fahrplan"]["id"]
+        except KeyError:
+            abort(422)
+
+        if not payload["is_master"]:
```

Comment:
```suggestion
        if not payload.get("is_master"):
```

Reply:
I don't like splattering code with lots of `.get()`s. If we fear c3voc might send us a webhook with a bunch of data missing, i'd rather wrap everything in `try..except: abort(400)` ...